src/components/article/ArticleForm.jsx

```diff
@@ -94,8 +94,9 @@ class ArticleForm extends React.Component {
 
     const { actionType } = this.props;
     if (actionType === 'create') {
-      this.postArticle(event);
+      return this.postArticle(event);
     }
+    return this.postArticle(event);
```

Comment:
Why the duplication? This line seems to be doing exactly what the statement within the `if` statement is doing.